src/main/kotlin/blackjack/service/BlackjackService.kt

```diff
@@ -33,12 +37,25 @@ class BlackjackService {
         }
     }
 
+    fun resultBlackjackGameMoney(players: List<Player>, dealer: Dealer): List<BlackjackGameMoneyResult> {
+        val result = mutableListOf<BlackjackGameMoneyResult>()
+
+        players.forEach { player ->
+            when (dealer.determineResult(player)) {
+                MatchResult.WIN -> result.add(BlackjackGameMoneyResult(player.name, -player.betAmount))
+                MatchResult.LOSE -> result.add(BlackjackGameMoneyResult(player.name, player.betAmount))
+                MatchResult.DRAW -> result.add(BlackjackGameMoneyResult(player.name, 0.0))
+            }
+        }
+        return result
```

Comment:
`map` 연산을 활용해봐도 좋을 것 같아요.
```suggestion
        return players.map {
            when(dealer.determineResult(it)) {
                MatchResult.WIN -> BlackjackGameMoneyResult(it.name, -it.betAmount)
                MatchResult.LOSE -> BlackjackGameMoneyResult(it.name, it.betAmount)
                MatchResult.DRAW -> BlackjackGameMoneyResult(it.name, 0.0)
            }
        }
```

Reply:
map 연산으로 수정하였습니다~!